src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkRuleOrCheckpoint.kt

```diff
@@ -5,4 +5,17 @@ import com.jetbrains.python.psi.PyTypedElement
 
 interface SmkRuleOrCheckpoint : SmkRuleLike<SmkRuleOrCheckpointArgsSection>, PyTypedElement {
     fun collectWildcards(): List<Pair<PsiElement, String>>
```

Comment:
Not clear whether `collectWildcards()` should use use all wildcards or only from defined sections. Some kdoc is required here.